magda-minion-broken-link/src/index.ts

```diff
@@ -5,15 +5,35 @@ import commonYargs from "@magda/minion-framework/dist/commonYargs";
 
 const ID = "minion-broken-link";
 
+const coerceJson = (json?: string) => {
```

Comment:
Wouldn't we rather have it crash and make it obvious that something is broken if we pass bad config?

Reply:
Yeah~ it should make more sense to let it crash~ 👍 
Updated to throw the error~